bots/dangerfile.js

```diff
@@ -14,7 +14,7 @@ const includes = require('lodash.includes');
 const {danger, fail, warn} = require('danger');
 
 // Fails if the description is too short.
-if (!danger.github.pr.body || danger.github.pr.body.length < 10) {
+if (!danger.github.pr.body || danger.github.pr.body.length < 50) {
```

Comment:
The minimum text expected by this file is already ~40 characters, so bumping this.